src/test/resources/org/jenkinsci/plugins/github/migration/MigratorTest/shouldLoadDataAfterStart/com.cloudbees.jenkins.GitHubPushTrigger.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<com.cloudbees.jenkins.GitHubPushTrigger_-DescriptorImpl plugin="github@1.13.0-SNAPSHOT"/>
```

Comment:
Please use the released version. Otherwise it may be hard to triangulate migration causes, etc.